pkg/cluster/spec/bindversion.go

```diff
@@ -28,7 +28,8 @@ func TiDBComponentVersion(comp, version string) string {
 		ComponentPushwaygate,
 		ComponentCheckCollector,
 		ComponentSpark,
-		ComponentTiSpark:
+		ComponentTiSpark,
+		ComponentTiKVCDC: // TiKV-CDC use individual version.
```

Comment:
Is there any compatibility requirement or the user can always use the latest version of TiKV-CDC regardless of the TiDB version?

Reply:
No requirement. User can always use the latest version by now, and we will maintain the compatibility.